tests/containers/rootless_podman.pm

```diff
@@ -63,7 +64,9 @@ sub run {
 }
 
 sub softfail_and_skip_on_bsc1182874 {
-    if (script_run("podman run alpine", timeout => 120) != 0) {
+    my $alpine = registry_url('alpine');
+    script_retry("podman pull $alpine", delay => 15, retry => 6);
+    if (script_run("podman run $alpine", timeout => 120) != 0) {
```

Comment:
in case you go on with the above line, does the `timeout` still make sense here?

Reply:
Maybe not, maybe something lower like 60?

Reply:
its up to your judge. not a big deal IMO. i am leaving on you.